openassessment/xblock/submission_mixin.py

```diff
@@ -29,6 +29,23 @@ class SubmissionMixin(object):
 
     """
 
+    ALLOWED_IMAGE_MIME_TYPES = ['image/gif', 'image/jpeg', 'image/pjpeg', 'image/png']
+
+    ALLOWED_FILE_MIME_TYPES = ['application/pdf'] + ALLOWED_IMAGE_MIME_TYPES
+
+    # taken from http://www.howtogeek.com/137270/50-file-extensions-that-are-potentially-dangerous-on-windows/
+    # and http://pcsupport.about.com/od/tipstricks/a/execfileext.htm
+    # left out .js and office extensions
+    FILE_EXT_BLACK_LIST = [
+        'exe', 'msi', 'app', 'dmg', 'com', 'pif', 'application', 'gadget',
+        'msp', 'scr', 'hta', 'cpl', 'msc', 'jar', 'bat', 'cmd', 'vb', 'vbs',
+        'jse', 'ws', 'wsf', 'wsc', 'wsh', 'scf', 'lnk', 'inf', 'reg', 'ps1',
+        'ps1xml', 'ps2', 'ps2xml', 'psc1', 'psc2', 'msh', 'msh1', 'msh2', 'mshxml',
+        'msh1xml', 'msh2xml', 'action', 'apk', 'app', 'bin', 'command', 'csh',
+        'ins', 'inx', 'ipa', 'isu', 'job', 'mst', 'osx', 'out', 'paf', 'prg',
+        'rgs', 'run', 'sct', 'shb', 'shs', 'u3p', 'vbscript', 'vbe', 'workflow',
+    ]
+
```

Comment:
@lamagnifica  Yes. This is the only place you need to change if you want to edit the blacklist.